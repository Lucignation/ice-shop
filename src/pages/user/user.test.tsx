import axiosInstance from '../../axios/index';

const fetchData = async () => {
  const res = await axiosInstance.get('/users/3');
  return res.data;
};

const mockData = fetchData();

test('the data is a single user', async () => {
  const data = await fetchData();
  expect(data).toMatchSnapshot(mockData);
});
