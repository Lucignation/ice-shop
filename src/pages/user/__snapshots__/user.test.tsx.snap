// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the data is a array of users 1`] = `
Object {
  "__v": 0,
  "address": Object {
    "city": "Cullman",
    "geolocation": Object {
      "lat": "40.3467",
      "long": "-30.1310",
    },
    "number": 86,
    "street": "Frances Ct",
    "zipcode": "29567-1452",
  },
  "email": "kevin@gmail.com",
  "id": 3,
  "name": Object {
    "firstname": "kevin",
    "lastname": "ryan",
  },
  "password": "kev02937@",
  "phone": "1-567-094-1345",
  "username": "kevinryan",
}
`;

exports[`the data is a si of users 1`] = `
Object {
  "__v": 0,
  "address": Object {
    "city": "Cullman",
    "geolocation": Object {
      "lat": "40.3467",
      "long": "-30.1310",
    },
    "number": 86,
    "street": "Frances Ct",
    "zipcode": "29567-1452",
  },
  "email": "kevin@gmail.com",
  "id": 3,
  "name": Object {
    "firstname": "kevin",
    "lastname": "ryan",
  },
  "password": "kev02937@",
  "phone": "1-567-094-1345",
  "username": "kevinryan",
}
`;

exports[`the data is a single user 1`] = `
Object {
  "__v": 0,
  "address": Object {
    "city": "Cullman",
    "geolocation": Object {
      "lat": "40.3467",
      "long": "-30.1310",
    },
    "number": 86,
    "street": "Frances Ct",
    "zipcode": "29567-1452",
  },
  "email": "kevin@gmail.com",
  "id": 3,
  "name": Object {
    "firstname": "kevin",
    "lastname": "ryan",
  },
  "password": "kev02937@",
  "phone": "1-567-094-1345",
  "username": "kevinryan",
}
`;
